medical_patient_dob/views/medical_patient_dob_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+ <record id="medical_patient_dob_view" model="ir.ui.view">
+        <field name="name">medical.patient.dob.view</field>
+        <field name="model">medical.patient</field>
+        <field name="inherit_id" ref="medical.medical_patient_view_search" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='birthdate_date']" position="replace">
+                <field name="birthdate" />
```

Comment:
Where is medical core located? Is it not OCA/vertical-medical? I was overriding this line in this view: https://github.com/OCA/vertical-medical/blob/258297e095ab181ef76cc31ee2d8e90079a4012b/medical/views/medical_patient.xml#L115